scripts/send-trace-to-jaeger/src/main.rs

```diff
@@ -0,0 +1,89 @@
+use reqwest::blocking::Client;
+use serde_json::{Map, Number, Value};
+use std::env::args;
+use std::fs::File;
+use std::io::{self, BufRead};
+use std::path::Path;
+
+/// Read individual lines from a file.
+fn read_lines<P>(filename: P) -> io::Result<io::Lines<io::BufReader<File>>>
+where
+    P: AsRef<Path>,
+{
+    let file = File::open(filename)?;
+    Ok(io::BufReader::new(file).lines())
+}
+
+/// Log the url to view the trace in the browser.
+fn log_web_url(jaeger_web_ui_url: &str, trace_id: &str) {
+    println!(
+        "Jaeger trace will be available on {}/trace/{}",
+        jaeger_web_ui_url.to_string(),
+        trace_id.to_string()
+    )
+}
+
+/// Send trace JSON to Jaeger using ZipKin API.
+fn send_json_to_zipkin(zipkin_api: &str, value: &str) {
```

Comment:
I think it's better to accept `value: String`, as this function unconditionally converts `value` to `String`.